cli/kaos_cli/commands/build.py

```diff
@@ -76,6 +76,17 @@ def build(backend: BackendFacade, cloud, env, force, verbose, yes, local_backend
             click.style("Info", bold=True, fg='green'),
             click.style(cloud, bold=True, fg='red')))
 
+    # validate unused port for DOCKER
+    if cloud == DOCKER:
```

Comment:
any reason to not join the two booleans?

```suggestion
    if cloud == DOCKER and not validate_unused_port(80):
```

Reply:
No practical reason, I just separated them for readability, if you think they need to be merged, then I will merge them.

Reply:
Done.